core/src/por/api.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+import { OraklError, OraklErrorCode } from '../errors'
+import { CHAIN, ORAKL_NETWORK_API_URL } from '../settings'
+import { IAggregator } from '../types'
+import { buildUrl } from '../utils'
+import { IData } from './types'
+
+export async function loadAggregator({ aggregatorHash }: { aggregatorHash: string }) {
```

Comment:
@bayram98 If I understand it correctly, we are not going to activate the aggregator. Am I right?

Reply:
Yes.